samples/client/echo_api/java/apache-httpclient/src/test/java/org/openapitools/client/CustomTest.java

```diff
@@ -220,7 +220,7 @@ public void testDefaultValuesSerializationWithEmptyPayload() throws IOException
         Assert.assertNull(d.getArrayStringNullable());
         Assert.assertEquals(d.getArrayString().size(), 0);
 
-        Assert.assertEquals(apiClient.getObjectMapper().writeValueAsString(d), "{\"array_string_enum_ref_default\":[\"success\",\"failure\"],\"array_string_enum_default\":[\"success\",\"failure\"],\"array_string_default\":[\"failure\",\"skipped\"],\"array_integer_default\":[1,3],\"array_string\":[],\"array_string_nullable\":{\"present\":false},\"array_string_extension_nullable\":{\"present\":false},\"string_nullable\":{\"present\":false}}");
+        Assert.assertEquals(apiClient.getObjectMapper().writeValueAsString(d), "{\"array_string_enum_ref_default\":[\"success\",\"failure\"],\"array_string_enum_default\":[\"success\",\"failure\"],\"array_string_default\":[\"failure\",\"skipped\"],\"array_integer_default\":[1,3],\"array_string\":[]}");
```

Comment:
didn't want to change all the assertions.. but aren't those assertEquals params inverted?
expected should be the first param and the actual value right after/second param.

Reply:
you're right: https://junit.org/junit4/javadoc/4.8/org/junit/Assert.html#assertEquals(java.lang.Object,%20java.lang.Object)

feel free to submit another PR to correct it. thanks for pointing it out.